Sources/Showcase/FocusStatePlayground.swift

```diff
@@ -18,15 +19,43 @@ struct FocusStatePlayground: View {
                .focused($focusEnum, equals: .textA)
             TextField("textB", text: $textB)
                 .focused($focusEnum, equals: .textB)
+            
+            ChildTextField(
+                title: "textC",
+                value: $textC,
+                focusedField: $focusEnum,
+                assignedFocusField: FocusField.textC
+            )
+            
             Button("Set focus to A via Boolean") { focusBool = true }
             Button("Set focus to A via Enum") { focusEnum = .textA }
             Button("Set focus to B via Enum") { focusEnum = .textB }
+            Button("Set focus to C via Enum") { focusEnum = .textC }
             Button("Nil Enum") { focusEnum = nil }
         }
     }
 }
 
+struct ChildTextField: View {
+    
+    let title: String
+    @Binding var value: String
+    
+    #if !SKIP
+    @FocusState.Binding var focusedField: FocusField?
+    #else
+    @Binding var focusedField: FocusField?
+    #endif
```

Comment:
@aabewhite Do you think we could just typealias `FocusState.Binding` to `Binding` in SkipUI? I'm unclear on why `FocusState.Binding` exists at all.